.github/workflows/go.yml

```diff
@@ -30,5 +34,20 @@ jobs:
             dep ensure
         fi
 
-    - name: Build
-      run: go build -v .
+    - name: Build for linux/amd64
+      env:
+        GO_VERSION: v1.14
+        TIME: "${{ steps.current-time.outputs.time }}"
+      run: GOOS=linux GOARCH=amd64 go build -v -o bin/kadok -ldflags "-X main.Version=$GITHUB_REF -X main.BuildDate=$TIME -X main.GitCommit=$GITHUB_SHA -X main.GoVersion=$GO_VERSION"
+
+    - name: Build for windows/amd64
```

Comment:
is it require to build it for win ?

Reply:
Indeed, the pipeline generates 1 binary for Linux and 1 for Windows.